src/main/java/org/springframework/data/aerospike/cache/AerospikeCacheManager.java

```diff
@@ -63,9 +63,10 @@
 	 * set name "aerospike".
 	 * 
 	 * @param aerospikeClient the {@link AerospikeClient} instance.
+	 * @param aerospikeConverter
 	 */
-	public AerospikeCacheManager(AerospikeClient aerospikeClient) {
-		this(aerospikeClient, Collections.<String>emptyList());
+	public AerospikeCacheManager(AerospikeClient aerospikeClient, MappingAerospikeConverter aerospikeConverter) {
```

Comment:
Not in maven doesn't mean nobody is referencing the piece of code. :) Would it be more convenient to provide a converter unless necessary?